contracts/pcv/ERC20HoldingPCVDeposit.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import {PCVDeposit} from "./PCVDeposit.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/// @title ERC20HoldingPCVDeposit
+/// @notice PCVDeposit that is used to hold ERC20 tokens as a safe harbour. Deposit is a no-op
+contract ERC20HoldingPCVDeposit is PCVDeposit {
+    /// @notice the underlying token of the PCV deposit
+    IERC20 public token;
+
+    /// @notice a flag for whether to report the balance as protocol owned FEI
+    bool public isProtocolFeiDeposit;
+
+    constructor(
+        address _core,
+        IERC20 _token,
+        bool _isProtocolFeiDeposit
+    ) CoreRef(_core) {
+        token = _token;
+        isProtocolFeiDeposit = _isProtocolFeiDeposit;
```

Comment:
isProtocolFei should never really be used, there will be no FEI instance of this contract because there isn't a need to just hold dead FEI in a contract for anything

Reply:
Yep makes sense. This PR is WIP and was throwing things down before refining